src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -136,6 +136,19 @@ public IBlockState onBlockPlaced(World worldIn, BlockPos pos, EnumFacing facing,
         return this.getDefaultState().withProperty(FACING, enumFacing);
     }
 
+    //We unfortunately have to implement these two, to rotate our blocks in the structures.
+    @Override
+    public IBlockState withRotation(final IBlockState state, final Rotation rot)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)